src/app/dataValidator/rules/__snapshots__/topstories.test.js.snap

```diff
@@ -0,0 +1,14521 @@
+            },
+            "indexImage": Object {
+              "altText": "KeyFrame",
+              "copyrightHolder": "BBC",
+              "height": 549,
+              "href": "http://c.files.bbci.co.uk/66A2/production/_106347262_p075wbrp.jpg",
+              "id": "106347262",
+              "path": "/cpsprodpb/66A2/production/_106347262_p075wbrp.jpg",
+              "subType": "index",
+              "width": 976,
+            },
+            "locators": Object {
+              "assetUri": "/thai/features-47856329",
+              "cpsUrn": "urn:bbc:content:assetUri:thai/features-47856329",
+            },
+            "media": Object {
+              "advertising": true,
+              "caption": "งานวิจัยชี้แมวรู้จักชื่อตัวเอง",
+              "embedding": true,
+              "format": "video",
+              "id": "p075wbb5",
+              "imageCopyright": "BBC",
+              "imageUrl": "ichef.bbci.co.uk/images/ic/$recipe/p075wbrp.jpg",
+              "subType": "clip",
+              "synopses": Object {
+                "long": null,
+                "medium": null,
+                "short": "งานวิจัยชี้แมวรู้จักชื่อตัวเอง",
+              },
+              "title": "งานวิจัยชี้แมวรู้จักชื่อตัวเอง",
+              "versions": Array [
+                Object {
+                  "availableTerritories": Object {
+                    "nonUk": true,
+                    "uk": true,
+                  },
+                  "duration": 58,
+                  "types": Array [
+                    "Original",
+                  ],
+                  "versionId": "p075wbb9",
+                },
+              ],
+              "warnings": Object {},
+            },
+            "options": Object {
+              "isBreakingNews": false,
+              "isFactCheck": false,
+            },
+            "passport": Object {
+              "campaigns": Array [
+                Object {
+                  "campaignId": "5a988e1739461b000e9dabf6",
+                  "campaignName": "WS - CS",
+                },
+                Object {
+                  "campaignId": "5a988e2939461b000e9dabf8",
+                  "campaignName": "WS - Educate me",
+                },
+              ],
+              "category": Object {
+                "categoryId": "http://www.bbc.co.uk/ontologies/applicationlogic-news/News",
+                "categoryName": "News",
+              },
+            },
+            "section": Object {
+              "name": "รายงาน",
+              "subType": "IDX",
+              "type": "simple",
+              "uri": "/thai/features",
+            },
+            "summary": "งานวิจัยล่าสุดจากญี่ปุ่นพบหลักฐานว่า แมวบ้านอาจเข้าใจภาษามนุษย์มากกว่าที่เราคิด",
+            "timestamp": 1554778180,
+            "type": "cps",
+          },
+          Object {
+            "cpsType": "MAP",
+            "headlines": Object {
+              "headline": "จะเกิดอะไรขึ้นเมื่อยาปฏิชีวนะใช้ไม่ได้ผลอีกต่อไป",
+        
```

Comment:
Do we really need 14521 lines of snapshot? 😨

Reply:
I think so, it doesn't hurt and means if we change this logic we know it wont have negative side effects. Its more like a integration test

Reply:
They are needed, just so longggggggg